cylc/flow/scheduler.py

```diff
@@ -650,7 +650,8 @@ async def run_scheduler(self) -> None:
             # Non-async sleep - yield to other threads rather than event loop
             sleep(0)
             self.profiler.start()
-            await self.main_loop()
+            while True:  # MAIN LOOP
+                await self._main_loop()
```

Comment:
On changing to this, it might make sense (clarity reasons) to move the sleep code from the end of the `_main_loop` method and into the `while` block?

Reply:
Reasonable. I think something (besides the sleep) is using `tinit` which might be why I left it.

Reply:
True. In that case, fine to leave as-is.